src/functionalTest/resources/scenarios/RIA-3545-valid-200-response-error-code-2000.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "RIA-3545 Mock 200 response with generic error",
+  "disabled": true,
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "X00002000"
```

Comment:
I had to add functional test for each kind of error scenario and we do not have test data from Home Office (only 2 were provided). Also, Srihari has to test the scenario on HMCTS end and for this, I added certain HO references on the ia-home-office-mock-api
I created functional test and ran it against mock-api, then disabled it as it will fail when we connect to PRP1. The idea is when Home Office provides input for other error scenario, we can enable them with the right HO reference data.